.travis.yml

```diff
@@ -10,10 +10,11 @@ env:
   - PIPENV_IGNORE_VIRTUALENVS=1
 
 install:
-  - pip install pipenv
+  - pip install pipenv black
   - pipenv install --dev --deploy
 
 script:
+  - black --diff --check idunn/ tests/
```

Comment:
It could help to list directories we want to format in the pyproject.toml, we would only need to run `black .` and it will be easier to use with an IDE.